From 1c5cf237cedb3ee07ae8da5546e52d04edd9e4af Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Thu, 19 Nov 2020 12:19:37 +0000
Subject: [PATCH] Build fix for cuda

---
 python-package/setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/python-package/setup.py b/python-package/setup.py
index 4e3242f..1a7421d 100644
--- a/python-package/setup.py
+++ b/python-package/setup.py
@@ -124,6 +124,7 @@ def compile_cpp(use_mingw=False, use_gpu=False, use_cuda=False, use_mpi=False,
             cmake_cmd.append("-DOpenCL_LIBRARY={0}".format(opencl_library))
     elif use_cuda:
         cmake_cmd.append("-DUSE_CUDA=ON")
+        cmake_cmd.append("-DCMAKE_CUDA_HOST_COMPILER={}".format(os.environ["CMAKE_CUDA_HOST_COMPILER"]))
     if use_mpi:
         cmake_cmd.append("-DUSE_MPI=ON")
     if nomp:
-- 
1.8.3.1

