From eeb91f0e20434034164885b327f61eba20d1e424 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Sat, 9 Oct 2021 03:54:54 -0400
Subject: [PATCH] Fix os import in setup.py

---
 python-package/setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/python-package/setup.py b/python-package/setup.py
index 0570d06c..88dd6f73 100644
--- a/python-package/setup.py
+++ b/python-package/setup.py
@@ -4,6 +4,7 @@ import logging
 import struct
 import subprocess
 import sys
+import os
 from os import chdir
 from pathlib import Path
 from platform import system
-- 
2.32.0

